packages/storage/tsconfig.json

```diff
@@ -1,5 +1,4 @@
 {
   "extends": "../../tsconfig.json",
-  "files": ["./index.ts"],
-  "include": []
+  "files": ["./src/index.ts"]
```

Comment:
did you try after removing include: []? that's the fix i got from flopflip.

Reply:
Hmm, let me try again. I removed it because the sentry package also does not have it...

Reply:
Ooh ok, then we need to add it for each `tsconfig.json` I suppose. Good catch

Reply:
Wait, now I'm getting other errors 😱

Reply:
Ok this seems to fix it: https://github.com/commercetools/merchant-center-application-kit/pull/715/commits/19b7bb5500177e88c40040973190d86d649124f1